exercises/concept/tim-from-marketing-2/src/test/java/EmployeeServiceTest.java

```diff
@@ -0,0 +1,75 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Optional;
+
+// Annotation in order to use @BeforeAll in a non-static method
+@TestInstance(Lifecycle.PER_CLASS)
```

Comment:
I think this is causing the failure in the CI. It should work if you remove this and change the `@BeforeAll` on line 36 to `@BeforeEach`.

```suggestion
```